makeSpectral.py

```diff
@@ -0,0 +1,257 @@
+from os import listdir, rename
+from os.path import isfile, join, abspath, basename, splitext, dirname
+from typing import override
+from typing import TextIO, List
+from shutil import copy, SameFileError
+from argparse import ArgumentParser
+
+"""
+NOTE:
+	Although a little more cumbersome, I feel that providing the our
+	*replace substring* function extra verbosity is important.
+	This is because the User's chosen Name and Description
+	may occur in parts of the c++ code we do not want to edit.
+	The inclusion of the context around the needle we want to substitute
+	will help prevent accidental substitions.
+Happy Coding. Pix ❤️
+""" 
+
+class Cmd:
+	def __init__(self, p : str, n : str, d : str, v : bool):
+		self.path = p.pop() if p else None
+		self.name = n.pop() if n else None
+		self.description = d.pop() if d else None
+		self.verbose = v.pop() if v else False
```

Comment:
The behaviour of that cmd argument library is super weird. 🤷‍♂️ 
The fact that I'm having to pop() out of a singleton list is super weird in the first place.

I love your suggestion.
Because we have imported typing you could even do something like this (if you're inclined)
```python
self.verbose = v.pop() if type(v) is List[bool] else v
```